### DISCUSSION TITLE
Local drive and client independence

### DISCUSSION THREAD

**Discussion by 16pxdesign** [ltsp/ltsp#543] [Q&A]
Is that possible to store data on client local drive?
I`m asking mainly due fact - if I unplug ethernet cable, system go freeze till cable is not connected back

**Comment by alkisg** [comment]
You can mount /home from a local partition, for example by using an FSTAB_x line in ltsp.conf.
But even then, if you unplug the ethernet cable, and you run e.g. firefox, the system will freeze until you put the cable back, so that it can fetch /usr/bin/firefox from the server.
So if you want to be able to work after unplugging the ethernet cable, you'd need both the LTSP image and /home in a local partition.
If you manually put them there, then LTSP can use them; you would just use root=/dev/sdaX instead of root=/dev/nfs in ltsp.ipxe.
Also, after running `ltsp image` on the LTSP server, you'd need to re-send the new image to all the client disks.
There's no automation for this setup (yet), it needs to be done manually.
